docs/Symbols.md

```diff
@@ -0,0 +1,98 @@
+# Symbols
+
+## Symbols filter settings
+
+By default Outline (symbols) display `DOM elements`, `processing instruction` and `DTD element, entity, attribute list declarations`.
+
+The DOM attributes and text nodes are not displayed in the outline. Indeed to prevent from good performance, symbols must be limitated; displaying attributes and text nodes in the outline could generate a lot of symbols.
+
+Here a sample Outline with maven `pom.xml`:
+
+![Outline of pom.xml without filter](images/Symbols/SymbolsPOMWithoutFilter.png)
+
+In maven context, the text nodes information are important and we should prefer showing text nodes in the Outline:
+
+![Outline of pom.xml with text filter](images/Symbols/SymbolsPOMWithTextFilter.png)
+
+In Spring context, the @Id attribute information are important and we should prefer showing those attributes in the Outline:
+
+![Outline of Spring beans with @id filter](images/Symbols/SymbolsBeansWithIdAttrFilter.png)
+
+In other words, displaying attributes or text nodes depends on the XML file.
+
+`Symbols filters` gives the capability to define filter (to include or exclude) some attributes, some text nodes for a given XML file kind.
+
+Symbols filter are composed with:
+
+  * `pattern` (required) : it defines which XML files must use the filter.

```

Comment:
I would say :

a regular expression matching the file names to which this filter should apply.

Reply:
fixed